etc/pip-requires

```diff
@@ -19,3 +19,4 @@ numpy==1.14.2                                     # Numpy for forecasting T3C
 paramiko==2.6.0                                   # SSH2 protocol library
 Flask==1.1.1                                      # Python web framework
 M2Crypto<=0.35.2                                  # Dependency of FTS rest API; Temporary fix since 0.33 does not install on CC7
+oic==0.15.1                                       # for authentication via OpenID Connect protocol
```

Comment:
You use `0.15.1` because it is the last version supporting py2.7 I assume?

Reply:
I still use what I started with, which indeed supported py2.7. I would leave the upgrade for once this pull request is merged.